pages/get-started.tsx

```diff
@@ -142,17 +142,24 @@ export default function GetStartedPage() {
           title={content.steps[2].title}
           bodyText={content.steps[2].body}
         >
-          <Typography variant="h5">{content.steps[2].data[0]}</Typography>
-          <Grid container spacing={8} sx={{ my: 1 }}>
-            {content.rejectionReasons.map((fact) => (
+          <Grid container spacing={8} sx={{ mb: 8 }}>
+            {content.step3Substeps.map((step3Substep) => (
               <FactCard
-                className={`fact-card-${fact.id}`}
-                key={fact.id}
-                icon="none"
-                details={fact.details}
+                className={`fact-card-${step3Substep.id}`}
+                key={step3Substep.id}
+                title={step3Substep.id}
+                details={step3Substep.details}
+                sxProps={{ height: '100%', justifyContent: 'flex-start' }}
               />
             ))}
           </Grid>
+          <MuiMarkdown>
+            {content.steps[2].data[0]}
+          </MuiMarkdown>
+          <Typography variant="h5" sx={{ my: 4 }}>{content.steps[2].data[1]}</Typography>
+          <MuiMarkdown>
+            {content.rejectionReasons.details}
+          </MuiMarkdown>
```

Comment:
I can understand some of those `Markdown` uses but this one I believe we can have the same result using `Material UI` components.

Did you have any problem trying to not use `Markdown` here?